src/overlays/actors/ovl_Boss_Mo/z_boss_mo.c

```diff
@@ -3046,6 +3049,7 @@ void BossMo_DrawEffects(BossMoEffect* effect, PlayState* play) {
     CLOSE_DISPS(gfxCtx, "../z_boss_mo.c", 7482);
 }
 
+#if PLATFORM_GC
 void BossMo_Unknown(void) {
     // Appears to be a test function for sound effects.
     static Vec3f zeroVec = { 0.0f, 0.0f, 0.0f };
```

Comment:
unrelated to your changes, but the `sZeroVec` for the vector would be preferred. Or even `sSfxPos` cause it plays the sound effect from 0,0,0.
And also I wouldnt be mad about `BossMo_Unknown` -> `BossMo_SfxTest`